paymentsource/client.go

```diff
@@ -161,8 +161,8 @@ func (s Client) Verify(id string, params *stripe.SourceVerifyParams) (*stripe.Pa
 
 	if len(params.Customer) > 0 {
 		err = s.B.Call("POST", fmt.Sprintf("/customers/%v/sources/%v/verify", params.Customer, id), s.Key, body, &params.Params, source)
-	} else {
-		err = errors.New("Only customer bank accounts can be verified in this manner.")
+	} else if len(params.Values) > 0 {
+		err = s.B.Call("POST", fmt.Sprintf("/sources/%v/verify", id), s.Key, body, &params.Params, source)
 	}
```

Comment:
you're likely missing an else that raises an error no?

Reply:
👍  Added an `else` clause. The error message is a bit vague, I'm open to suggestions if you can think of something better!